src/components/Popups/PopupItem.tsx

```diff
@@ -7,25 +7,28 @@ import { PopupContent } from '../../state/application/reducer'
 import FailedNetworkSwitchPopup from './FailedNetworkSwitchPopup'
 import TransactionPopup from './TransactionPopup'
 
-const StyledClose = styled(X)`
+const StyledClose = styled(X)<{ padding: number }>`
   position: absolute;
-  right: 20px;
-  top: 20px;
+  right: ${({ padding }) => `${padding}px`};
+  top: ${({ padding }) => `${padding}px`};
 
   :hover {
     cursor: pointer;
   }
 `
-const Popup = styled.div`
+const Popup = styled.div<{ isTransactionPopup: boolean; show: boolean }>`
   display: inline-block;
   width: 100%;
-  padding: 1em;
+  opacity: ${({ show }) => (show ? '1' : '0')};
   background-color: ${({ theme }) => theme.backgroundSurface};
   position: relative;
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
   border-radius: 16px;
-  padding: 20px;
-  padding-right: 35px;
+  padding: ${({ isTransactionPopup }) => (!isTransactionPopup ? '20px' : '2px 0px')};
```

Comment:
Combine 27/28 into one line so 28 doesn't overwrite 27. 

Don't negate the conditional.

Reply:
done ty